prepare/metrics/grouped_instance_metrics.py

```diff
@@ -0,0 +1,362 @@
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 1.0},
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 1.0},
+    {"score": 1.0},
+    {"score": 0.0},
+    {"score": 0.0},
+    {"score": 0.0},
+]
+
+for instance in instance_targets_accuracy:
+    instance.update({"accuracy": instance["score"], "score_name": "accuracy"})
+
+metric = GroupMeanAccuracy()
+global_target = {
+    "group_mean_accuracy": 0.22,
+    "score": 0.22,
+    "score_name": "group_mean_accuracy",
+    "score_ci_low": 0.02,
+    "score_ci_high": 0.44,
+    "group_mean_accuracy_ci_low": 0.02,
+    "group_mean_accuracy_ci_high": 0.44,
+}
+
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=references,
+    instance_targets=instance_targets_accuracy,
+    global_target=global_target,
+    additional_inputs=additional_inputs,
+)
+
+add_to_catalog(metric, "metrics.group_mean_accuracy", overwrite=True)
+
+
+metric = GroupMeanStringContainment()
+global_target = {
+    "group_mean_string_containment": 0.49,
+    "score": 0.49,
+    "score_name": "group_mean_string_containment",
+    "score_ci_low": 0.16,
+    "score_ci_high": 0.71,
+    "group_mean_string_containment_ci_low": 0.16,
+    "group_mean_string_containment_ci_high": 0.71,
+}
+
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=references,
+    instance_targets=instance_targets_string_containment,
+    global_target=global_target,
+    additional_inputs=additional_inputs,
+)
+
+add_to_catalog(metric, "metrics.group_mean_string_containment", overwrite=True)
+
+
+# PDR
+metric = GroupPDRAccuracy()
+global_target = {
+    "group_pdr_accuracy": 0.83,
+    "score": 0.83,
+    "score_name": "group_pdr_accuracy",
+    "score_ci_low": 0.38,
+    "score_ci_high": 1.0,
+    "group_pdr_accuracy_ci_low": 0.38,
+    "group_pdr_accuracy_ci_high": 1.0,
+}
+
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=references,
+    instance_targets=instance_targets_accuracy,
+    global_target=global_target,
+    additional_inputs=additional_inputs,
+)
+
+add_to_catalog(metric, "metrics.group_pdr_accuracy", overwrite=True)
+
+
+metric = GroupPDRStringContainment()
+global_target = {
+    "group_pdr_string_containment": 0.44,
+    "score": 0.44,
+    "score_name": "group_pdr_string_containment",
+    "score_ci_low": 0.14,
+    "score_ci_high": 1.0,
+    "group_pdr_string_containment_ci_low": 0.14,
+    "group_pdr_string_containment_ci_high": 1.0,
+}
+
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=references,
+    instance_targets=instance_targets_string_containment,
+    global_target=global_target,
+    additional_inputs=additional_inputs,
+)
+
+add_to_catalog(metric, "metrics.group_pdr_string_containment", overwrite=True)
+
+# Try Cohen's h instead of PDR since is symmetric and defined when baseline is 0
```

Comment:
Not sure i understand this comment (?)

Reply:
It is just an alternative to the above metric which is undefined in many cases, and the CI and aggregates would ignore these in the computation, which is losing information.

Reply:
I can delete the comment, I just made an additional note in the class documentation.  It doesn't belong here

Reply:
ok, sounds good